.github/workflows/ci-tests/Examples_tests/test_launcher.sh

```diff
@@ -371,25 +451,36 @@ for dir in ./*/; do
 
 done # end non connected tests
 
-#****************************************************************************************************
-#*********************************** Start of Datc/s connected tests ********************************
-#****************************************************************************************************
+echo
+echo "****************************************************************************************************"
+echo "*********************************** Start of Datc/s connected tests ********************************"
+echo "****************************************************************************************************"
+echo
 
 erase_all_devices
 
 # Flash MAIN_DEVICE with BLE_datc
 cd $MSDK_DIR/Examples/$MAIN_DEVICE_NAME_UPPER/BLE_datc
+echo pwd=`pwd`
+echo
+
 make -j8
 
 # flash client first because it takes longer
 cd $MSDK_DIR/Examples/$MAIN_DEVICE_NAME_UPPER/BLE_datc/build
-printf "> Flashing BLE_datc on main device: $MAIN_DEVICE_NAME_UPPER\r\n "
-flash_with_openocd_fast $MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID 1
+printf "\r\n> Flashing BLE_datc on main device: $MAIN_DEVICE_NAME_UPPER\r\n"
+#flash_with_openocd_fast $MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID 1
+flash_with_openocd $MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID 1
 
 # flash DUT with BLE_dats
 cd $MSDK_DIR/Examples/$DUT_NAME_UPPER/BLE_dats/build
-printf "> Flashing BLE_dats on DUT $DUT_NAME_UPPER \r\n"
-flash_with_openocd_fast $DUT_NAME_LOWER $DUT_ID 2
+printf "\r\n> Flashing BLE_dats on DUT $DUT_NAME_UPPER\r\n"
+#flash_with_openocd_fast $DUT_NAME_LOWER $DUT_ID 2
+flash_with_openocd $DUT_NAME_LOWER $DUT_ID 2
+
+# Reset the two boards
+reset_board_by_openocd max32655 $MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID
+reset_board_by_openocd max32655 $DUT_NAME_LOWER         $DUT_ID
```

Comment:
if device under test is NOT max32655 does first argument still have to be max32655?

Reply:
also that function seems to use 2 args but we are giving 3

Reply:
Correct. The 2nd board is not always max32655.